client/lib/local-storage-bypass/test/index.js

```diff
@@ -1,28 +1,23 @@
-import { expect } from 'chai';
-import { spy } from 'sinon';
 import { key, clear, setItem, getItem, removeItem, length } from '..';
 
 describe( 'localstorage-bypass', () => {
 	// Spy on the original localStorage functions
-	const _setItem = spy();
-	const _getItem = spy( ( x ) => x );
-	const _removeItem = spy();
-	let db = {};
+	const _setItem = jest.fn();
+	const _getItem = jest.fn().mockImplementation( ( x ) => x );
```

Comment:
Could be `jest.fn( ( x ) => x)`

Reply:
Good point, cleaned up in https://github.com/Automattic/wp-calypso/pull/63751/commits/69f339fde26709b4e8d8bf5eeec112c1976fc084